lib/random/formatter.rb

```diff
@@ -369,4 +375,80 @@ def alphanumeric(n = nil, chars: ALPHANUMERIC)
     n = 16 if n.nil?
     choose(chars, n)
   end
+
+  # Default chunk size for #phrase
+  CHUNK_SIZE = 5
+
+  # Default punctuation joining chunks for #phrase
+  PUNCT = %w[! - + / . ^ ~].freeze
+
+  # Default excluded characters for #phrase
+  EXCLUDE = %w"0 O o 1 l".freeze
```

Comment:
Is there a reason for having a different separator around the lists? `%w[]` and `%w""`, I mean.